src/app/components/ReadTime/index.tsx

```diff
@@ -21,15 +23,13 @@ const DEFAULT_TRANSLATIONS = {
   minutes: 'minutes',
 };
 
-const ReadTime = ({
-  readTimeValue,
-  readTimeVariant,
-  className,
-}: ReadTimeProps) => {
-  const showReadTime = readTimeVariant && readTimeVariant !== 'off';
-  if (!showReadTime) return null;
-
+const GetTranslations = () => {
   const { translations } = use(ServiceContext);
+  return translations;
+};
+
+const processReadTime = (readTimeValue: number) => {
+  const translations = GetTranslations();
```

Comment:
Is this worth having in its own function, or do we just implement what's inside that function here? 
```suggestion
  const { translations } = use(ServiceContext);
```

Reply:
Yep - I was on the fence - though this way is less code. (I think we would normally pass Translations into this helper function, but I wanted to get the Translations in this shared function since we're doing it in two places)